### PR TITLE
Dob 744 Port SonarQube section

### PR DESCRIPTION
**Pull request by ChrisSchreiber**
Updated SonarQube instructions from confluence docs and port them here

### CODE REVIEW

File: docs/5/5.7-sonarqube.md [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+5. Generate a new token and make sure to save it.
+
+## Add SonarQube scanner to Jenkins
+1. SSH into your Jenkins host
+2. Install the SonarQube Scanner [Installation Instructions](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner)
+3. Open Jenkins web console [http://localhost:8080](http://localhost:8080)
+4. Install the SonarQube Scanner Jenkins Plugin [Installation Instructions](https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner+for+Jenkins)
+
+## Configure Jenkins job to run SonarQube scanner
+1. Goto Jenkins Global Tool Configuration and enter the path to where you installed the scanner
```
Comment (matt2m) [inline docs/5/5.7-sonarqube.md]: Insert a space in-between go to

File: docs/5/5.7-sonarqube.md [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+![](img/jenkins-sonarqube_scanner.png)
+2. Goto Jenkins Configure System and configure you SonarQube server
```
Comment (matt2m) [inline docs/5/5.7-sonarqube.md]: same here, include a space in-between go to

File: docs/5/5.7-sonarqube.md [hunk 3]
```diff
@@ -1,17 +1,44 @@
...
+![](img/jenkins-sonarqube_server.png)
+3. Create a Spring PetClinic Maven project and in the configure the SonarQube scanner to run as a Pre Step
```
Comment (matt2m) [inline docs/5/5.7-sonarqube.md]: period at the end

File: docs/5/5.7-sonarqube.md (hunk 2 above)
Comment (ChrisSchreiber) [reply docs/5/5.7-sonarqube.md]: fixed

File: docs/5/5.7-sonarqube.md (hunk 3 above)
Comment (ChrisSchreiber) [reply docs/5/5.7-sonarqube.md]: fixed

**Pull request merged**